CMSgov/easi-app
# [NOREF] Fixed the TRB request URL in the TRB request consult meeting and lead email links

# EASI-000

## Changes and Description

- Fixed the TRB request URL in the TRB request consult meeting and lead email links

## How to test this change

Passing automated tests should suffice here, but if you'd like to do a more thorough test, you can see the instructions in [this PR](https://github.com/CMSgov/easi-app/pull/1849)

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
